astropy/table/table.py

```diff
@@ -205,13 +208,16 @@ def as_array(self):
         return data
 
     def __init__(self, data=None, masked=None, names=None, dtype=None,
-                 meta=None, copy=True, rows=None):
+                 meta=None, copy=True, rows=None, copy_indices=True):
 
         # Set up a placeholder empty table
         self._set_masked(masked)
         self.columns = self.TableColumns()
         self.meta = meta
         self.formatter = self.TableFormatter()
+        self._copy_indices = True # copy indices from this Table by default
+        self._modify_indices = True # modify indices upon Table modification
```

Comment:
This was what I was previously using to create an indexing mode in which indices ignore modifications, but `_modify_indices` can now be deleted -- thanks for the catch.